.github/workflows/bypass-css-review.yaml

```diff
@@ -0,0 +1,34 @@
+name: Bypass CSS File Reviews by Admin
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  bypass-css-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up GitHub CLI Authentication
+        env:
+          GH_TOKEN: ${{ secrets.ADMIN_PAT }}
+        run: |
+          AUTHOR=$(gh pr view ${{ github.event.pull_request.number }} --json author --jq '.author.login')
+          ADMIN_CHECK=$(gh api user/memberships/orgs/scidsg --jq '.role')
+          if [ "$ADMIN_CHECK" != "admin" ]; then
+            echo "User is not an admin or is not a public member of the organization. Exiting."
+            exit 1
+          fi
+
+      - name: Check for CSS files and approve if conditions are met
+        run: |
+          if git diff --name-only main ${{ github.head_ref }} | grep -q '\.css$'; then
```

Comment:
I think instead of success on matching files, we should instead fail on non-matching,
```suggestion
          if ! git diff --name-only main ${{ github.head_ref }} | grep -v -q '\.css$'; then
```
This has the effect of auto-approving only when all the files modified are of the desired type.

Reply:
Wouldn't this mean that one workflow will always fail for most PRs?

![Screenshot 2024-09-03 at 9 56 10 AM](https://github.com/user-attachments/assets/d32efc69-8f3e-44de-b48a-4672aa77f568)

Reply:
![Screenshot 2024-09-03 at 9 58 12 AM](https://github.com/user-attachments/assets/08e2fe6b-8b53-49ca-97c9-a617f0e8828f)

Reply:
I updated to use a non-failing exit